index.html

```diff
@@ -83,7 +83,7 @@
       ></a-entity>
 
       <a-entity id="player"
-        camera="userHeight: 1.6"
```

Comment:
This was taken out because the new version of A-frame warned that property `userHeight` didn't exist. I'm not sure if I should replace it.